### PR TITLE
Update readme.md to add more detailed docs for getting started

### PR DESCRIPTION
**Pull request by matthewbentley**
Getting started info.  Will be useful if/when somebody else takes over maintenance of this.  These are the steps I took to get the development server running.

I will work on getting this running on the ACM server (via docker) when I have some free time next week.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,26 @@ For people who have rusty rails, follow the following steps to setup.
 - Require the library `require "sis_importer"`
 - Import SIS Data: `SISImporter.import_sis`
 
+### Getting started
+1. Clone: `git clone https://github.com/hacsoc/the_jolly_advisor.git`
+2. New branch: `git checkout -b my-branch`
+3. Install rvm: https://rvm.io
+4. Switch to Ruby 2.2.0  
+ a. `rvm install ruby-2.2.0`  
+ b. `rvm use 2.2.0`
```
Comment (ajm188) [inline README.md]: When you cd into the project root, rvm will detect and set the ruby version from the Gemfile, so this line isn't totally necessary, unless you have a line in your .rvmrc to disable that behavior

File: README.md [hunk 2]
```diff
@@ -22,6 +22,26 @@ For people who have rusty rails, follow the following steps to setup.
...
+ a. `su - postgres; createuser --interactive` use your system username when it asks for role to add  
+ b. `createdb the_jolly_advisor_development`  
+7. `cd /path/to/the_jolly_advisor`
+8. Install gems: `bundle`
+9. Migrate db: `rake db:migrate` this might take a few minutes
+10. Open rails console; get SIS data  
+ a. `rails c`  
+ b. `require “sis_importer”`  
+ c. `SISImporter.import_sis`  
+11. Start local server: `rails server`
```
Comment (ajm188) [inline README.md]: `rails s`. It's aliased, just like rails console -> rails c

Comment (ajm188) [inline README.md]: Also might want to note that the server is listening on port 3000 by default. So to see stuff in your browser, you should go to localhost:3000

**Comment by matthewbentley** [comment]
Will fix and re-submit

**Comment by ajm188** [comment]
For what it's worth, I'll be sticking around to help maintain this for at least a year, so in addition to this readme (which is much better), people can open up issues on the repo and I'll see them. Maybe tag them with a "question" tag or something.

**Pull request closed without merging**